index.js

```diff
@@ -211,14 +210,14 @@ RedisClient.prototype.do_auth = function () {
     self.send_anyway = true;
     self.send_command("auth", [this.auth_pass], function (err, res) {
         if (err) {
-            if (err.toString().match("LOADING")) {
+            if (/LOADING/.test(err.toString())) {
```

Comment:
@simontabor err.message is very likely better but as far as I know a precompiled regex with .test is faster than .indexOf and it's at least more readable in my opinion. It's difficult to write a proper test to compare the regex against the .indexOf as v8 is very likely inlining everything right away.